File: 01_programming_basics/03_more_exercises/01_first_steps_in_coding/06_fishland.py
price_mackerel_kg = float(input())
price_toy_kg = float(input())
bonito_kg = float(input())
horse_mackerel_kg = float(input())
mussels_kg = float(input())
price_bonito_kg = price_mackerel_kg * 1.60
price_horse_mackerel_kg = price_toy_kg * 1.80
price_mussels = 7.50
total = bonito_kg * price_bonito_kg + horse_mackerel_kg * price_horse_mackerel_kg + mussels_kg * price_mussels
print(f"{total:.2f}")

File: 05_JavaScriptBasics/03_MoreExercises/01_FirstStepsInCoding/06_fishland.js
function fishland(input){
    let priceMackerelKg = Number(input[0])
    let priceSprinkleKg = Number(input[1])
    let kgBonito = Number(input[2])
    let kgSafrid = Number(input[3])
    let kgMussels = Number(input[4])

    let priceBonitoKg = priceMackerelKg * 1.60
    let priceSafridKg = priceSprinkleKg * 1.80
    let priceMussels = 7.50

    totalSum = priceBonitoKg * kgBonito + priceSafridKg * kgSafrid + priceMussels * kgMussels

    console.log(totalSum.toFixed(2))
}
